### DISCUSSION TITLE
Add htsfile and tabix from base htslib

### DISCUSSION THREAD

**Discussion by RandyHarr** [sandbox-bio/sandbox.bio#14] [Q&A] [answered]
Thank you for restoring using local files.  Might I additionally suggest adding htsfile and tabix to the list of available tools.  This is to allow people to do simple tasks on local files they have.  Many web-based tools people try to use on WGS files (*iobio.io, genvue, etc) require certain formats. Sometimes users do not know if they have those files in the right format. htsfile helps with that.  Tabix is just another tool, like with samtools and bcftools, that can be used to create missing index files. The sandbox.io playground is a stepping stone to learning about command line and bioinformatic tools before they dive into a full install themselves.

Of course tabix is only useful as any of the indexing tools are IF you can download the file to your desktop as well.  htsfile is useful no matter what to help identify file format and content.  I constantly use it as a quick and easy look at compressed (with or without extensions indicating as such) FA reference models.  95% are simply gzip compressed but have to be bgzip compressed to be indexed and used.  htsfle differentiates the compression method it finds.

**Comment by robertaboukhalil** [comment, marked as answer]
`htsfile` and `tabix` should now be available!

**Comment by RandyHarr** [comment]
You are the "bomb"!   Now we need to develop a tutorial on using your site for those having done personal / consumer WGS tests and maybe needed to do simple manipulation of their delivered files without installing any tools.  A great resource now available.

**Reply by robertaboukhalil** [reply]
Haha thanks!
